inputs/test/test_containers.py

```diff
@@ -46,7 +46,7 @@ def test_containers_local_inputs_integration(tmp_path, inputs_module):
             ["podman", "inspect", "-f", "{{ .Id }}", base_tag],
             universal_newlines=True).strip()
         inputs = {
-            "type": "org.osbuild.containers",
+            "type": "org.osbuild.containers-storage",
```

Comment:
Just one small question regarding this. This is set first as `org.osbuild.containers` in the previous commit and then changed to `org.osbuild.containers-storage` here, is this because of the class name of the input?

Reply:
This is a great catch and a good question - my original test was wrong here and then I fixed that but that is of course quite confusing. I'm sorry for that, I changed it now so that the original test is already correct.

Reply:
Excellent catch again! Thank you! Fixed.